src/components/hooks/HookDetails.tsx

```diff
@@ -0,0 +1,185 @@
+import React from 'react';
+
+import { createFragmentContainer } from 'react-relay';
+import { createStyles, WithStyles, withStyles } from '@material-ui/core/styles';
+import { RouteComponentProps, withRouter } from 'react-router-dom';
+import { graphql } from 'babel-plugin-relay/macro';
+import { HookDetails_hook } from './__generated__/HookDetails_hook.graphql';
+import { Helmet as Head } from 'react-helmet';
+import { Card, CardContent } from '@material-ui/core';
+import RepositoryNameChip from '../chips/RepositoryNameChip';
+import BuildBranchNameChip from '../chips/BuildBranchNameChip';
+import BuildChangeChip from '../chips/BuildChangeChip';
+import TaskNameChip from '../chips/TaskNameChip';
+import HookCreatedChip from '../chips/HookCreatedChip';
+import Typography from '@material-ui/core/Typography';
+import Logs from '../logs/Logs';
+import HookStatusChip from '../chips/HookStatusChip';
+import CirrusFavicon from '../common/CirrusFavicon';
+import classNames from 'classnames';
+import { useNotificationColor } from '../../utils/colors';
+
+const styles = theme =>
+  createStyles({
+    gap: {
+      paddingTop: theme.spacing(2),
+    },
+    chip: {
+      marginTop: theme.spacing(0.5),
+      marginBottom: theme.spacing(0.5),
+      marginRight: theme.spacing(0.5),
+    },
+    wrapper: {
+      padding: 0,
+      display: 'flex',
+      flexWrap: 'wrap',
+    },
+    io: {
+      padding: theme.spacing(1),
+      background: '#212121',
+      color: '#FFFFFF',
```

Comment:
Can we use some colors from the theme? Similar to `Logs.tsx` 🤔

Reply:
See https://github.com/cirruslabs/cirrus-ci-web/pull/355/commits/e007b900503391c87069f210e22d7d2d53aa2c41.